src/engine/Filter.h

```diff
@@ -56,6 +56,15 @@ class Filter : public Operation {
       }
       if (_type == SparqlFilter::FilterType::NE) {
         return _subtree->getSizeEstimate();
+      } else if (_type == SparqlFilter::FilterType::PREFIX) {
+        // strip the leading ^ and " and assume each character gets rid of 10
+        // percent of the entries
```

Comment:
// Assume that only 10^-k entries remain, where k is the length of the prefix. The reason for the -2 is that at this point, _rhs always starts with ^"